_posts/2020-03-09-our-core-values.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Our Core Values 
+date: 2020-03-09 14:30:00
+categories: ["values"]
+author: etagwerker
+
+---
+
+At [Ombu Labs](https://www.ombulabs.com) we have many values that have been key
+to our success. This is an article about five core values that differentiate 
+our company from the rest.
+
+Every team member is expected to follow these values, especially when things get 
+tough. This is a living document: It's open source and open to enhancements by 
+design. We have been tweaking these values ever since I started the company.
+
+<!--more-->
+
+## Team First
+
+We can't run a successful consultancy if we don’t value and respect every team 
+member. We don't want people who "live to work". Workaholism is toxic and it 
+leads to burnout. We strive to maintain a healthy work/life balance, to "work 
+to live" a great life, and to grow as professionals within the company.
+
+In terms of our communication, we adhere to the contributor covenant v2.0, not 
+just for our open source projects, but also for any interaction we have with 
+team members: [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md)
```

Comment:
@etagwerker The link is broken

Reply:
This is a big topic - workaholism is a big problem, as are unrealistic expectations all around.  I think it's good that we take a stance on it.